scripts/templates/template.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Template
+parameters:
+- name: IMAGE_TAG
+  required: true
+- name: REPO_NAME
+  value: REPLACED_BY_SCRIPT
+  required: true
+metadata:
+  name: selectorsyncset-template
+objects:
+# replace old SSS w/ nothing so we don't have manual cleanup to do
```

Comment:
Everything past line #12 in this file needs to be deleted.  This repo doesn't have some legacy named SSS deployed that needs clobbered.